scripts/release-helm.sh

```diff
@@ -50,8 +50,8 @@ scripts/helm-dependencies.sh validate
 # Read the shared scripting config.
 source "distribution/helm/scripting-config.sh"
 
-# Filter out vector and aggregator if not publishing to nightly
-NIGHTLY_CHARTS=( "vector" "vector-aggregator" )
+# Filter out vector if not publishing to nightly
+NIGHTLY_CHARTS=( "vector" )
```

Comment:
Just to remind me, this `vector` chart deploys both?

Reply:
Correct - wanted to make sure our versioning and everything worked as we wanted before releasing those as stable